frontend/src/pages/ProjectDetails.tsx

```diff
@@ -80,117 +67,44 @@ const ProjectDetailsPage = () => {
         message="Sorry, the project you're looking for doesn't exist"
       />
     )
+  const projectDetails = [
+    { label: 'Last Updated', value: formatDate(project.updated_at) },
+    { label: 'Level', value: project.level[0].toUpperCase() + project.level.slice(1) },
+    { label: 'Organization', value: project.organizations },
+    { label: 'Project Leaders', value: project.leaders.join(', ') },
+    { label: 'Type', value: project.type[0].toUpperCase() + project.type.slice(1) },
+    {
+      label: 'URL',
+      value: (
+        <a href={project.url} className="hover:underline dark:text-sky-600">
+          {project.url}
+        </a>
+      ),
+    },
+  ]
 
+  const projectStats = {
+    Contributors: project.contributors_count,
+    Forks: project.forks_count,
+    Issues: project.issues_count,
+    Repositories: project.repositories_count,
+    Stars: project.stars_count,
+  }
```

Comment:
I'm not sure if this is needed -- we have this type of data for many entities -- projects/chapters/committees.

Reply:
Also it should be lowercased.